plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsEvents.java

```diff
@@ -40,10 +40,11 @@
   // Metadata keys
   public static final String APP_ENGINE_LOCAL_SERVER_MODE = "mode";
   public static final String APP_ENGINE_NEW_PROJECT_WIZARD_TYPE = "type";
-  public static final String APP_ENGINE_NEW_PROJECT_WIZARD_BUILD_TOOL = "build_tool";
+  public static final String APP_ENGINE_NEW_PROJECT_WIZARD_BUILD_TOOL = "build.tool";
```

Comment:
I think it's OK to change this. We haven't set up a graph for this yet.

Reply:
Oh, actually, [this was merged 13 days ago](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2955/files#diff-90f045e3a2a727c3315674098de0cabcR43) and [1.6.0B was tagged 26 days ago](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/releases/tag/v1.6.0B), so we can freely change this.